utils/audio.js

```diff
@@ -1,18 +1,101 @@
 'use strict';
 
-module.exports = {
-  buildSfxCollection(game, componentName, clipCount) {
-    // FIXME: Make this a class.
+class Audio {
```

Comment:
Yeah partly! I also thought it would make sense as most of the methods require the game object, so I just stuck that into a constructor instead. What do you reckon?